ruby-challenges.rb

```diff
@@ -9,7 +9,16 @@
 # Expected output: '42 is even'
 num3 = 221
 # Expected output: '221 is odd'
-
+def even_or_odd value
+    if value == value.odd? || value.even? 
```

Comment:
Just running a conditional with value.odd? would be enough for the evaluation to come back as true and return the value is odd.